git_theta/updates/sparse.py

```diff
@@ -12,6 +13,13 @@
 class SparseUpdate(IncrementalUpdate):
     """An update where only some parameters are touched."""
 
+    def __init__(
+        self, serializer: params.Serializer, threshold: Optional[float] = 1e-12
```

Comment:
`thereshold` shouldn't be `Optional` it is always used.

Reply:
Changed it to always used